### DISCUSSION TITLE
[Feature Request] Dev Collab Request + Dev interest list

### DISCUSSION THREAD

**Discussion by DarkyAndreas** [RetroAchievements/RAWeb#1192] [Ideas]
Dev Collab Request: Basically the dev who wants to collab clicks the "Make collab claim" button on the site, the main dev gets a notification that someone would like to collab on a set with them and the main dev can either accept or decline the request. Currently everyone could make a collab claim without the permission of the main dev

Dev interested list: A list similar to 'Set Requests' but for developers who are interested to develop a set for the game, could be a nice feature for collabs. However, I thik this list should only be visible by developers (and maybe Jr. Devs)

**Comment by televandalist** [comment]
We were actually discussing a dev interest feature recently since it has worked pretty well with how we do it during the rollouts. I think it would work best if devs could leave a short comment alongside it. Like if I want to mark something I'm interested in, but would only like to do a specific game mode or something.

**Reply by StingX2** [reply]
I think its great to hang it out there as an option for the die hards just not to rely on it solely when a rollout starts. PS2, and megaduck rollouts had issues that caused delays and when it came time for claim lock ins a bunch of tastes changed. There were absolutes (mostly the rpg crowd) but otherwise many people started bouncing.

**Comment by Jamiras** [comment]
I see two separate feature requests here:
* Dev interest list: See also #731 and #799. It would just be another subtype.
* Collaboration confirmation: This seems reasonable.
